csob/api.py

```diff
@@ -234,9 +256,12 @@ def payment_close(self, pay_id: str, total_amount: Optional[AmountHundredths]) -
         Returns:
             APIResponse - Return values are identical with the definition contained in the payment/init operation.
         """
-        raise NotImplementedError()
+        if type(total_amount) == Decimal:
```

Comment:
isinstance

Reply:
proč tam může přijít něco jiného než decimal?

Reply:
Ve specifikaci API je, že se posílají setiny jako celé číslo tak jsem si říkal, že to někdo může chtít ukládat v DB už jako celá číslo setin.

Reply:
to bych asi nedovoloval, Nechal bych to poslat jen v jednom formátu

Reply:
a to isinstance?

Reply:
již opraveno